addon/components/action-button.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember'
+
+export default Ember.Component.extend({
```

Comment:
I think it's better in this case to extend FrostButton and override onClick - see the point example for this in ember-frost-object-browser

This also protects us from being a proxy layer

If you do this you can also delete the template for this component